system/Helpers/test_helper.php

```diff
@@ -30,7 +30,7 @@ function fake($model, ?array $overrides = null, $persist = true)
     {
         $fabricator = new Fabricator($model);
 
-        if ($overrides) {
+        if ($overrides !== null && $overrides !== []) {
```

Comment:
cast to array

Reply:
Only `$overrides !== null` is sufficient.

Reply:
I think the other cases are almost the same, but the intent is `null` means "do nothing", `array` means to override with it. Or I think the true type is non-empty-array or null.

Reply:
updated to ony check !==null